requirements.txt

```diff
@@ -2,3 +2,4 @@ peft
 accelerate
 transformers
 trl
+pytest
```

Comment:
`pytest` is a dev requirement not a production requirement. Please remove it.

Reply:
then should I create a requirements-dev.txt? 
Also should I include mypy with pylint in it?